src/shared/components/dateRangePicker/NewDatePicker.tsx

```diff
@@ -220,9 +229,187 @@ const DatePicker: FC = () => {
     setIsDatePickerOpen(prev => !prev)
   }
 
+  return (
+    <Dropdown.Menu
+      style={{
+        width: isDatePickerOpen ? 669 : 400,
+      }}
+      className="date-picker--menu"
+      maxHeight={367}
+    >
+      <FlexBox direction={FlexDirection.Row} alignItems={AlignItems.Stretch}>
+        {isDatePickerOpen && (
+          <div className="react-datepicker-ignore-onclickoutside date-picker--calendar-dropdown">
+            <div className="date-picker__select-date-picker range-picker--date-pickers">
+              <InputLabel className="date-picker--label__calendar">
+                Calendar
+              </InputLabel>
+              <ReactDatePicker
+                calendarClassName="range-picker--calendar"
+                dateFormat="yyyy-MM-dd HH:mm"
+                dayClassName={dayClassName}
+                disabledKeyboardNavigation
+                fixedHeight
+                inline
+                selectsRange
+                shouldCloseOnSelect={false}
+                startOpen
+                startDate={startDate}
+                endDate={endDate}
+                onChange={handleSelectDate}
+              />
+            </div>
+          </div>
+        )}
+        <div className="date-picker--calendar-dropdown">
+          <div className="date-picker__select-time-range">
+            <InputLabel className="date-picker--label">
+              Select time range
+              <QuestionMarkTooltip
+                className="date-picker--question-mark"
+                diameter={18}
+                color={ComponentColor.Primary}
+                tooltipContents={
+                  <>
+                    Use a relative duration (now(), -1h, -5m),{'\n'}
+                    absolute time (2022-08-28 14:26:00),{'\n'}
+                    or integer (Unix timestamp in seconds,{'\n'}
+                    like 1567029600).&nbsp;
+                  </>
+                }
+                tooltipStyle={{maxWidth: 285}}
+              />
+            </InputLabel>
+            <Form.Element
+              className="date-picker--form"
+              label="From"
+              errorMessage={inputStartErrorMessage}
+              required
+            >
+              <Input
+                className="date-picker__input"
+                onChange={handleSetStartDate}
+                status={
+                  inputStartErrorMessage === NBSP
+                    ? ComponentStatus.Default
+                    : ComponentStatus.Error
+                }
+                type={InputType.Text}
+                value={inputStartDate}
```

Comment:
```suggestion
                value={inputStartDate || ''}
```
Just an edge case, there is a dev warning when click on the time range picker before selecting any bucket. Adding an empty string should remove the dev warning